### PR TITLE
update index

### PR DESCRIPTION
**Pull request by AlejandroAlt**
Corrección ortográfico, cambio de palabra "theme"

## Description

Hi, i´m developer and want to contribute, so here's a list of the changes proposed to the index part of the tutorial.

1. Punctuation mark
2. Grammar corrections

### CODE REVIEW

File: docs/tutorial/index.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ disableTableOfContents: true
...
-5.  ["Source plugins" y renderizar la data extraida](/tutorial/part-five/): Usa un "source plugin" para extraer datos de tus post en Markdown a tu sitio web y crea una pagina "index" con la lista de tus posts.
+5.  ["Source plugins" y renderizar la data extraida](/tutorial/part-five/): Usa un "source plugin" para extraer datos de tus post en Markdown a tu sitio web y crea una página "index" con la lista de tus posts.
 6.  ["Transformer plugins"](/tutorial/part-six/): Usa un "transformer plugin" para transformar tus posts de Markdown a una forma que tu blog pueda renderizar.
-7.  [Crea páginas programáticamente desde data](/tutorial/part-seven/): Aprende cómo create páginas programáticamente para tus blog posts.
+7.  [Crea páginas programáticamente desde data](/tutorial/part-seven/): Aprende cómo crear páginas programáticamente para tus blog posts.
 8.  [Prepara si sitio web para publicarlo](/tutorial/part-eight/): Aprende como auditar el performance de tu sitio web y buenas prácticas de accesibilidad, SEO y más.
 
-## Tutoriales sobre "Themes"
+## Tutoriales sobre "Temas"
```
Comment (jackbravo) [inline docs/tutorial/index.md]: I would leave this without quotes.

**Comment by horacioh** [comment]
Thank you so much @AlejandroAlt & @jackbravo!!

like I said to @mattdark on #1, if you want, we can setup a roadmap/workflow so we can coordinate the translation here until the Gatsby team figures it out how the will want to translate?

I would be more than happy to work with you on this!.

thanks again! 👍

**Comment by jackbravo** [comment]
Thanks a lot for the quick response @horacioh!

We are working on a contribution sprint for gatsby as part of the ccoss.org event (Summit for Open Source Contributors in Mexico).

Maybe something similar to how gatsby gives github group access to any one who gets a PR merged? That is, giving write access to this repo to any person that gets a PR merged?

We are also trying to add everyone to the discord gatsby server, so we can follow up discussions there.

**Comment by horacioh** [comment]
that would be amazing!

I will do this manually now, but I'm open to make this more automated :)

**Comment by horacioh** [comment]
@AlejandroAlt @jackbravo @mattdark

you all should have an invitation to this repo.

let's use github also to manage pending translations and in progress? that would help avoiding doubling work on translations :)

any suggestions?

**Comment by jackbravo** [comment]
We are suggesting creating PRs even for in-progress translations. At least for the work we manage to finish as part of the summit. That way someone else can pick it up later.

**Comment by horacioh** [comment]
sounds good!

I saw you are doing a workshop on contributing to gatsby. that's great!
let me know how I can help and if there's anything else you need to make contributions easier to manage.

**Pull request merged**